run.py

```diff
@@ -12,7 +12,7 @@
 
 from numpy.random import seed, shuffle
 
-from tensorflow import set_random_seed
```

Comment:
tensorflow 2.0 moved the set_random_seed.py to tensorflow.random and renamed it to just set_seed.